alacritty_terminal/src/ansi.rs

```diff
@@ -320,11 +320,11 @@ pub trait Handler {
     /// Reset an indexed color to original value.
     fn reset_color(&mut self, _: usize) {}
 
-    /// Set the clipboard.
-    fn set_clipboard(&mut self, _: u8, _: &[u8]) {}
+    /// Request to set a clipboard.
+    fn request_clipboard_set(&mut self, _: u8, _: &[u8]) {}
 
-    /// Write clipboard data to child.
-    fn write_clipboard<W: io::Write>(&mut self, _: u8, _: &mut W, _: &str) {}
+    /// Request to paste to a child.
+    fn request_clipboard_write(&mut self, _: u8, _: &str) {}
```

Comment:
Since this is just a trait, I think the old names actually fit a bit better. The only thing I don't like is the name `set_` vs `write_`, since it's kinda confusing which one does which.

I believe I've actually changed this on my pager PR:
https://github.com/chrisduerr/alacritty/blob/pager/alacritty_terminal/src/ansi.rs#L323-L327

I think those names would make it clearer what is actually going on.